### PR TITLE
Refactor space layout

### PR DESCRIPTION
**Pull request by bonustrack**

### CODE REVIEW

**Comment by ChaituVR** [comment]
Looks very nice 👌 

I have few thoughts:

- Strategies list can be hyperlinked to a particular strategy page or would be cool if we can display the strategy model on the proposal page
  image
  image


- Would be nice to add Twitter, Github links in the about section (maybe Network and plugins too? 🤔)
- Core and Comunity are not needed anymore?
  image
  if yes we can remove this field?
  image

File: src/components/Block/Space.vue [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+          class="d-block px-4 py-2 sidenav-item"
+        />
+        <router-link
+          :to="{ name: 'about' }"
+          v-text="$t('about')"
+          :class="$route.name === 'about' && 'router-link-exact-active'"
+          class="d-block px-4 py-2 sidenav-item"
+        />
+        <router-link
+          v-if="isAdmin"
```
Comment (ChaituVR) [inline src/components/Block/Space.vue:35]: Not related to these pr changes, but can we also add isOwner here? I have no admins to my space and I can't see this 😋

Comment (bonustrack) [reply src/components/Block/Space.vue:35]: Query for a space owner take too long to resolve, i'd prefer not adding it.

**Comment by hongnhut199** [comment]
> No description provided.

**Comment by samuveth** [comment]
Awesome! I think this is a good base that we can start improving more in follow up PR's. One thing that is a little confusing is that when you click back on delegate it goes back to the home page.

**Comment by bonustrack** [comment]
@ChaituVR "Show only member proposals" field is still being used. When it's "on", only members can post a proposal, if "off" anyone can post (if they have minScore)

File: src/views/Space.vue [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+    // Infinite scroll with pagination
+    const {
+      loadBy,
+      limit,
+      loadingMore,
+      stopLoadingMore,
+      loadMore
+    } = useInfiniteLoader();
+
+    useScrollMonitor(() =>
```
Comment (samuveth) [inline src/views/Space.vue:82]: @bonustrack if there aren't any major changes you are looking to make in this PR I will start a PR where a query with Apollo and move this into a composable that can be used on timeline also.

Comment (bonustrack) [reply src/views/Space.vue:82]: I will merge this PR first then you can do that change with composable

**Pull request merged**